### PR TITLE
Bug 1839967 - Updated the Firefox Browser imagery assets.

### PR DESCRIPTION
**Pull request by t-p-white**
Icons provided by @cwzilla 

⚠️ **Updated** 
Updated assets optimized using imageOptim.

- Enabled lossy minification
- PNG quality 80% (default)
- Optimization level: Extra

Release APK size diff: +2.2KB
Screenshot 2023-09-05 at 11 21 02

Beta APK size diff: +2.1KB
Screenshot 2023-09-05 at 11 22 45

Nightly APK size diff: +37KB
Screenshot 2023-09-05 at 11 22 04

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1839967

### CODE REVIEW

**Review by cwzilla — APPROVED** [review]

**Comment by t-p-white** [comment]
> Could you add the file size diff to the root comment?
>
> Curious that a good chunk of them are larger after the "Browser" text removal

Thanks @MozillaNoah, I've updated the details on the root comment. I can't explain the size differences for the icons, perhaps @cwzilla can provide some context?

**Comment by cwzilla** [comment]
> > Could you add the file size diff to the root comment?
> > Curious that a good chunk of them are larger after the "Browser" text removal
> 
> Thanks @MozillaNoah, I've updated the details on the root comment. I can't give any insight into the size differences for the icons, perhaps @cwzilla can provide some context?

Hmm I'm not entirely sure either. I didn't run them through an optimizer or anything, could that be it?

**Comment by gabrielluong** [comment]
Is there some sort of PNG optimizer we can run these icons through? I recall in the past that we probably did this. @Amejia481 do you have any memory or recollection here? We can reach out to the general slack to see if anyone else remembers.

I think we should investigate whether or not we can optimize these icons before landing since we want to avoid increasing APK size unwantedly especially when we are actually reducing the amount of visuals around existing icons.

File: fenix/app/src/nightly/res/drawable/ic_wordmark_text_normal.png
Comment (t-p-white) [inline fenix/app/src/nightly/res/drawable/ic_wordmark_text_normal.png:1]: Needs height set to 117px

File: fenix/app/src/nightly/res/drawable/ic_wordmark_text_private.png
Comment (t-p-white) [inline fenix/app/src/nightly/res/drawable/ic_wordmark_text_private.png:1]: Needs height set to 117px

**Comment by t-p-white** [comment]
@gabrielluong found the tool used to compress PNGs was `imageoptim`

**Comment by Amejia481** [comment]
> PNG optimizer

Unfortunately, I only remember that we were doing optimizations for `VectorDrawable`s using 🥑 [avocado](https://github.com/alexjlockwood/avocado) and [svgo](https://github.com/svg/svgo).

**Comment by t-p-white** [comment]
The updated PNG assets have been optimized using `imageOptim`. There is a very small increase of 2.3KB in the Release APK. Rather than further reducing the quality of the PNG through optimization, I think we should investigate converting these (along with others) to the WebP format.

**Review by MozillaNoah — APPROVED** [review]
I'm cool with landing these for now and filing a follow-up to investigate the webp conversion (and linking this conversation there).

**Review by cwzilla — APPROVED** [review]

**Pull request merged**